### ISSUE TITLE
A11y: Aria labels input refactoring

### ISSUE THREAD

**Issue by ffriedl89** [dynatrace-oss/barista#79]
We need to deprecate all the aria labels like `aria-label-close` that are violating our accessibility tests and rename the inputs to ariaLabelClose so when they are bound they are in the template.
